ios/Framework/LiferayScreens.xcodeproj/xcshareddata/xcschemes/LiferayScreens.xcscheme

```diff
@@ -66,9 +67,10 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
```

Comment:
What do you think about creating a new target for building the framework needed for Xamarin? (including disable bitcode) This way we don't need to worry about setting the build configuration to debug again (which compiles faster) :)

Reply:
Great idea!